src/main/java/entity/cards/VileWeave.java

```diff
@@ -0,0 +1,49 @@
+package entity.cards;
+
+import static entity.EntityMod.makeCardPath;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import entity.EntityMod;
+import entity.characters.Entity;
+import entity.powers.FluxPower;
+//Vile weave	common	skill	1	Block 10(13). apply 1 flux to yourself
```

Comment:
Leave a space between import and comment.
```suggestion

//Vile weave	common	skill	1	Block 10(13). apply 1 flux to yourself
```